packages/tldraw/src/lib/ui/components/DebugMenu/DefaultDebugMenuContent.tsx

```diff
@@ -269,7 +269,6 @@ const DebugFlagToggle = track(function DebugFlagToggle({
 	return (
 		<TldrawUiMenuCheckboxItem
 			id={flag.name}
-			title={flag.name}
```

Comment:
out of curiousity: why the removal of `title` here (and in the lang menu)?

Reply:
This component doesn't actually do anything with that prop